UpdatedSyntheticDataset/SyntheticDataset2/ImageCreator/modular_target.py

```diff
@@ -0,0 +1,92 @@
+from PIL import Image
```

Comment:
A lot of these classes are similar. Please create abstract base classes and extend them with the different options.